pkg/operators/kubemanager/types/types.go

```diff
@@ -1,27 +1,17 @@
-// Copyright 2019-2022 The Inspektor Gadget authors
+// Copyright 2025 The Inspektor Gadget authors
 //
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//     http://www.apache.org/licenses/LICENSE-2.0
+//	http://www.apache.org/licenses/LICENSE-2.0
 //
 // Unless required by applicable law or agreed to in writing, software
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package common
+package types
 
-const (
-	MaxContainersPerNode = 1024
-	NameMaxLength        = 256
-)
-
-type Container struct {
-	ContainerID [NameMaxLength]byte
-	Namespace   [NameMaxLength]byte
-	Pod         [NameMaxLength]byte
-	Container   [NameMaxLength]byte
-}
+const DefaultSocketFile = "/run/hook-service.socket"
```

Comment:
nit: make it explicit this is for the hook service. `DefaultHookServiceSocketFile`